sh/tests/division_test.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+# Unofficial Bash Strict Mode
+set -euo pipefail
+IFS='
+'
+
+script_dirS=$(cd -P "$(dirname "$BASH_SOURCE")" &>/dev/null && pwd)
+
+challenge() {
+    submitted=$(bash "$script_dirS"/student/division.sh $1 $2)
+    expected=$(bash "$script_dirS"/solutions/division.sh $1 $2)
+
+    diff <(echo "$submitted") <(echo "$expected")
+}
+
+challenge "10" "2"
+challenge "4" "2"
+challenge "0.5" "0.5"
+challenge "5" "2"
+challenge "0.5"
```

Comment:
This is not testing properly it gives this error
`./division_test.sh: line 11: $2: unbound variable`

Reply:
Thanks. I added a condition to test with one argument when one is provided and with two when there is two.